src/main/kotlin/org/wycliffeassociates/otter/jvm/app/widgets/takecard/events/PlayOrPauseEvent.kt

```diff
@@ -0,0 +1,12 @@
+package org.wycliffeassociates.otter.jvm.app.widgets.takecard.events
+
+import javafx.event.Event
+import javafx.event.EventType
+
+class PlayOrPauseEvent(type: EventType<PlayOrPauseEvent>) : Event(type) {
+
+    companion object {
+        val PLAY: EventType<PlayOrPauseEvent> = EventType("PLAY")
+        val PAUSE: EventType<PlayOrPauseEvent> = EventType("PAUSE")
+    }
```

Comment:
This can be reworked if you decide not to pass types into your event constructors.  Possibly PlayOrPauseEvent becomes a sealed class, or maybe you add a PlayPause enum that is passed into the constructor.